lms/djangoapps/ccx/views.py

```diff
@@ -166,10 +168,19 @@ def dashboard(request, course, ccx=None):
         schedule = get_ccx_schedule(course, ccx)
         grading_policy = get_override_for_ccx(
             ccx, course, 'grading_policy', course.grading_policy)
+        enrolled_students = CourseEnrollment.objects.filter(course_id=ccx_locator, is_active=True)
         context['schedule'] = json.dumps(schedule, indent=4)
         context['save_url'] = reverse(
             'save_ccx', kwargs={'course_id': ccx_locator})
-        context['ccx_members'] = CourseEnrollment.objects.filter(course_id=ccx_locator, is_active=True)
+        context['ccx_members'] = (
+            enrolled_students
+            if not configuration_helpers.get_value('HIDE_MASTER_COURSE_STAFF_FROM_STUDENT_LIST', False)
+            else exclude_master_course_staff_users(
```

Comment:
If we agreed to remove `exclude_master_course_staff_users` usage related to CCXs, I think something like this might do the job, off course we would have to address that in a `run_extension_point`.
```suggestion
            else LicensedEnrollments.objects.filter(courseenrollment__in=enrolled_students)
```